playground/centernet.res18.coco.512size/train_net.py

```diff
@@ -0,0 +1,126 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
```

Comment:
Such a file is duplicated with tools/train_net.py, or you should consider combine them together

Reply:
Ok, I will try use the same train way as on linux